.gitignore

```diff
@@ -63,14 +63,13 @@ venv.bak/
 
 # created files
 lists.json
-lists-only.json
 out.json
 id2iri_*
 stashed*
 **/~$*.*
 testdata/tmp/
 testdata/test-list.json
-knora/dsplib/docker/sipi.docker-config.lua
+src/dsp_tools/docker/sipi.docker-config.lua
```

Comment:
I think `dsp_tools` counts as package and packages should not contain underscores. See https://peps.python.org/pep-0008/#package-and-module-names

Reply:
You're right. 

So I tried to rename it to `dsptools`, but ran into a number of difficulties with the git submodule. It's not fun renaming a parent of a git submodule... I don't know how I managed to do it before Christmas: then I suceeded to rename `knora/dsplib` to `src/dsp_tools`. I don't know what I did wrong today, but I ruined my clone of dsp-tools so badly that I will have to make a fresh clone. And I hope you understand that I don't want to try it a second time. After all, PEP8 only *discourages* underscores in package names.

Do you think we can leave it as it is?